cmd/importcorpora/main.go

```diff
@@ -0,0 +1,203 @@
+// Usage: go run cmd/importcorpora/main.go
+//
+// Updates the at the bottom of this file specified data files with content from dariusk/corpora.
+package main
+
+import (
+	"archive/zip"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// URL to zip file
+const corporaArchive = "https://github.com/dariusk/corpora/archive/master.zip"
+
+// Path to write Go file to
+const targetDir = "pkg/data"
+
+// Content of a Go file
+const fileTemplate = `package data
+
+var %s = %s`
```

Comment:
It's probably a good idea to add a comment here because the variable is exported and linters would complain... not a big deal but something like `// %s is an array of %s` should suffice